### ISSUE TITLE
Keyboard shortcut to translate a page via linguist

### ISSUE THREAD

**Issue by t3dium** [translate-tools/linguist#360]
Would be a very convenient feature to have

**Comment by vitonsky** [comment]
Duplicate of #266

This feature already implemented, check section "Page translation" in preferences

![image](https://github.com/translate-tools/linguist/assets/86191922/52615523-b63d-4f98-825d-f8ee21711940)

**Comment by t3dium** [comment]
> Duplicate of #266
> 
> This feature already implemented, check section "Page translation" in preferences
> 
> ![image](https://user-images.githubusercontent.com/86191922/241965733-52615523-b63d-4f98-825d-f8ee21711940.png)

Oh thats good, missed that option in settings. Im not sure if this is a bug but ctrl, shift and alt dont appear when trying to record a hotkey on firefox for me. Though i was able to get ``/  '`` working, which works fine for me. This also only affects firefox (i have arkenfox's userjs installed if thats relevant), and worked on a chromium based browser.

**Comment by t3dium** [comment]
Yeah this seems to only occur when ``privacy.resistFingerprinting`` (included in arkenfox's userjs), is set to True on a firefox browser. Otherwise keybindings including ctrl/shift/alt, are able to be registered

**Comment by vitonsky** [comment]
It looks the problem could be solved with handle hotkeys by special WebExtension API.
I will keep in mind this problem and will fix it if we will have similar problems with shortcuts implemented on content script side.

Thank you for report, it's are worth contribution to problems knowledge base
